api/agent/config.go

```diff
@@ -122,10 +127,12 @@ const (
 func NewConfig() (*Config, error) {
 
 	cfg := &Config{
-		MinDockerVersion: "17.10.0-ce",
-		MaxLogSize:       1 * 1024 * 1024,
-		PreForkImage:     "busybox",
-		PreForkCmd:       "tail -f /dev/null",
+		MinDockerVersion:        "17.10.0-ce",
+		MaxLogSize:              1 * 1024 * 1024,
+		PreForkImage:            "busybox",
+		PreForkCmd:              "tail -f /dev/null",
+		MaxImageCacheSize:       0,
+		ImageCacheCleanInterval: 0,
```

Comment:
minor nit here, this is initialized to zero, which we don't need to do in go-lang. But below interval is defaulted to 5 mins. Could you pls clean it up?